pladder/snusk.py

```diff
@@ -36,7 +45,10 @@ def example_snusk(self, a, b):
         return self._format_quad(quad)
 
     def _format_quad(self, quad):
-        return "{}{} i {}{}".format(*quad)
+            prep = random.choice(self._prep_entries)
+            quad.insert(2, prep) #Insert the preposition into the quad which technically makes it a pent, but who keeps score? 
```

Comment:
Ok, so let's stop calling it a quad then... :-) How about "parts"? I can change i.